web/src/scheduled_messages.js

```diff
@@ -180,7 +180,8 @@ export function get_filtered_send_opts(date) {
     const send_times = compute_send_times(date);
 
     const day = date.getDay(); // Starts with 0 for Sunday.
-    const hours = date.getHours();
+    // Construct an HHMM integer of the time for comparison below
+    const hhmm = date.getHours() * 100 + date.getMinutes(); // e.g., 1806 at 6:06pm
```

Comment:
I think I have the same high-level comment here as in https://github.com/zulip/zulip/pull/25476; it'd be nice to have the "5 minutes" be a constant defined at the top of the file, and we should figure out how to make the tests verify the new logic. Might make sense to just move this to be another commit in that PR, or the other way around, so they can share the refactoring.

Reply:
That makes sense. I think I'll just move these changes over to #25476. I'll close out this PR when I've done so.